src/components/Buttons/__tests__/__snapshots__/Button.spec.js.snap

```diff
@@ -561,9 +561,7 @@ exports[`Button renders correctly when disabled 1`] = `
   <button
     data-radium={true}
     disabled={true}
-    onBlur={[Function]}
     onClick={[Function]}
-    onFocus={[Function]}
```

Comment:
🤔 Interesting. Seems like Radium was somehow adding these to the output by having a `:focus` sudo style.

Reply:
Yeah, I was surprised that removing
```
':focus': {	
    outline: 'none'	
  }
```
had this impact on the snap shot.